redaxo/src/core/lib/console/assets_sync.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ * 
+ * @author staabm
+ *
+ * @internal
+ */
+class rex_command_assets_sync extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Sync folders and files of /assets with /redaxo/src/addons/my-addon/assets (or plugin) respectively /redaxo/src/core/assets folders');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output);
+
+        foreach(rex_package::getInstalledPackages() as $package) {
+            if ($package instanceof rex_addon) {
+                $assetsPublicPath = $package->getAssetsPath();
+                $assetsSrcPath = $package->getPath('assets/');
+            } else {
+                // we assume "plugin" assets folder is a subfolder of the "addon" assets folder
+                // and will therefore already be synced when handling the addon
+                continue;

```

Comment:
Das hatten wir irgendwo weiter oben schon in der Diskussion, die Plugin-Assets liegen separat im Plugin-Ordner, und müssen somit auch separat gesynct werden.
Bei der Installation eines Plugins wird
`redaxo/src/addons/my-addon/plugins/my-plugin/assets` kopiert nach
`assets/addons/my-addon/plugins/my-plugin`.

Somit ist hier keine Unterscheidung zwischen Addons/Plugins notwendig/sinnvoll aus meiner Sicht, beide müssen gleich behandelt werden (mit `$package->getAssetsPath()` und `$package->getPath('assets/')`).

Reply:
> der kommentar bezieht sich auf /assets/addons/meinAddon/plugins/meinPlugin/ d.h. sobald der addon folder synchronisiert ist, sind automatisch auch die plugins darunter synchronisiert.

Reply:
Ahh jetzt verstehe ich was du meinst...

Reply:
Bitte schau dir die Pfade mal ganz genau an, die Plugin-Assets in src liegen woanders, als wo du sie gerade (implizit) hin synchronisierst.
Beispiel: https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/be_style/plugins/customizer/assets/css/styles.css

Versuche die mal in beide Richtungen zu synchronisieren.